potodo/_github.py

```diff
@@ -33,14 +33,23 @@ def get_repo_name() -> str:
 
 
 def get_reservation_list() -> Mapping[str, str]:
-    """Will get the repository name then request all the issues and put them in a dict
     """
-    # Gets the issues into a dict
-    issues = requests.get(
-        "https://api.github.com/repos/" + get_repo_name() + "/issues"
-    ).json()
-    # Creates the issue dict with the name and the login
-    reservations: dict = {
-        issue["title"].split()[-1].lower(): issue["user"]["login"] for issue in issues
-    }
+    Will get the repository name then request all the issues and put them in a dict
+    """
+
+    issues: list = []
+    next = "https://api.github.com/repos/" + get_repo_name() + "/issues?state=open"
+    while next:
+        resp = requests.get(next)
+        issues += issues
```

Comment:
Semantically it's never clear if += means append or extend, so I try to avoid it, but you have the right to keep it, I bet it's very personal.

Reply:
Well in that case it's both lol